pkg/tagger/collectors/docker_main.go

```diff
@@ -104,7 +105,7 @@ func (c *DockerCollector) processEvent(e *docker.ContainerEvent) {
 	case "die":
 		info = &TagInfo{Entity: e.ContainerEntityName(), Source: dockerCollectorName, DeleteEntity: true}
 	case "start", "rename":
-		low, orchestrator, high, standard, err := c.fetchForDockerID(e.ContainerID)
+		low, orchestrator, high, standard, err := c.fetchForDockerID(e.ContainerID, e.Action == "start")
```

Comment:
nit: Maybe it's time to define an Enum for the different docker events filter keys and values